scripts/init-docker.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
```

Comment:
Something like this https://github.com/dotnet/buildtools/commit/de8abc7225c27f0b7dfd35d72deee19f7f672721 ?